src/serac/physics/solid.hpp

```diff
@@ -351,6 +362,24 @@ class Solid : public BasePhysics {
   virtual const serac::FiniteElementState& solveAdjoint(mfem::ParLinearForm& adjoint_load_form,
                                                         FiniteElementState*  state_with_essential_boundary = nullptr);
 
+  /**
+   * @brief Compute the implicit sensitivity of the quantity of interest used in defining the load for the adjoint
```

Comment:
It would probably be worth mentioning in the documentation here and for `bulkModulusSensitivitiy` that `solveAjoint` needs to be called first, and on the correct `adjoint_load_form`.